tox.ini

```diff
@@ -9,6 +9,8 @@ deps =
     drf38: djangorestframework<3.9
     drf39: djangorestframework<3.10
     drflatest: djangorestframework
+    celery44: -r{toxinidir}/requirements/celery44.txt
+    celery50: celery>=5.0,<6.0
```

Comment:
Ideally we'd also have a `celery50.txt`, but I suppose we can defer that for now.